test/nodes/test_document_classifier.py

```diff
@@ -22,6 +22,21 @@ def test_document_classifier(document_classifier):
         assert doc.to_dict()["meta"]["classification"]["label"] == expected_labels[i]
 
 
+@pytest.mark.integration
+def test_document_classifier_details(document_classifier):
+
+    docs = [
+        Document(content="""That's good. I like it.""", meta={"name": "0"}, id="1"),
+        Document(content="""That's bad. I don't like it.""", meta={"name": "1"}, id="2"),
```

Comment:
I don't think you need to specify IDs, you can remove them.

```suggestion
        Document(content="""That's good. I like it.""", meta={"name": "0"}),
        Document(content="""That's bad. I don't like it.""", meta={"name": "1"}),
```